website/docs/installing-podman-desktop/index.md

```diff
@@ -0,0 +1,406 @@
+---
+sidebar_position: 2
+title: Installing
+description: You can install Podman Desktop on Windows, macOS, and Linux.
+pagination_prev: null
+pagination_next: onboarding-for-containers/index
+keywords: [podman desktop, containers, podman, installing, installation]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Installing Podman Desktop
+
+<Tabs groupId="operating-systems" queryString="current-os">
+
+<TabItem value="win" label="Windows">
+
+<Tabs groupId="alternatives" queryString="current-installation-method">
+
+<TabItem value="installer" label="Windows installer">
+
+#### Prerequisites
+
+- Windows 10 or 11.
```

Comment:
Having these "prerequisites" simply list the OS seems odd. The tab / label says Windows? This is just repeating itself.